### DISCUSSION TITLE
Dropbox Tutorial

### DISCUSSION THREAD

**Discussion by hpposch** [elsa-workflows/elsa-core#1491] [Q&A]
Hi all!
I saw that there is a Dropbox Activity available.
But i have no idea how to configure it correctly.
Could anyone please provide a small guide on how to integrate it proberly in the code and what else is needed.

Thans in advance!
Heinz Peter

**Comment by swuecho** [comment]
also the File Activity

**Comment by ukepogluradr** [comment]
hi,

for example, you can use the FileExists activity in this way:

` public void Build(IWorkflowBuilder builder) =>
            builder
            .WriteLine("started ...")        
            .Then<FileExists>(setup=>setup.WithPath("c:\\errorlog.txt"), fileResult => {
                fileResult.When(OutcomeNames.True).WriteLine("founded").ThenNamed("end");
                fileResult.When(OutcomeNames.False).WriteLine("not founded").ThenNamed("end");
            })       
            .WriteLine("stopped ...").WithName("end");`


Don't forget adding AddFileActivities() statement in the Service collection. Like this:

`var services = new ServiceCollection()
                .AddElsa(options => options
                    .AddConsoleActivities()
                    .AddFileActivities()                  
                    .AddWorkflow<HelloWorld>())
                .BuildServiceProvider();`

**Reply by swuecho** [reply]
possible to use FileActivity in UI Desiginer?

**Reply by sfmskywalker** [reply]
That's certainly possible. All activities that can be used from code can also be used automatically from the designer, with a few small explicitly excluded exceptions such as `Workflow`, `Inline` and `CompositeActivity` (they will become hidden in a future update of Elsa).
